project/Dependencies.scala

```diff
@@ -471,6 +471,12 @@ object Dependencies {
       )
   )
 
+  val Text = Seq(
+    libraryDependencies ++= Seq(
+      "com.ibm.icu" % "icu4j" % "67.1"
```

Comment:
Do you know which license applies to this?
How "big" is it, would it make sense to require users to add it explicitly by marking it `provided`?

Reply:
According to maven, the licences is: https://raw.githubusercontent.com/unicode-org/icu/master/icu4c/LICENSE
Which, admittedly, I have not read.

Looking at https://mvnrepository.com/artifact/com.ibm.icu/icu4j/67.1, icu4j itself has no additional dependencies, which is good, and the entire api seems reasonably small: https://unicode-org.github.io/icu-docs/apidoc/released/icu4j/

Given that, I don't think I would want to mark it as provided, but maybe there is argument to be made.

Reply:
The license is OK for us. The jar is 12 MB in size. Alpakka Text is 23 kB.
I'm OK with depending on it, people concerned about library sizes can exclude it if needed.